brainframe/cli/frozen_utils.py

```diff
@@ -0,0 +1,52 @@
+import sys
+from pathlib import Path
+
+
+def is_frozen() -> bool:
+    """
+    :return: True if the application is running as a frozen executable
+    """
+    return getattr(sys, "frozen", False)
+
+
+RELATIVE_TRANSLATIONS_PATH = Path("brainframe/cli/translations")
+RELATIVE_DEFAULTS_FILE_PATH = Path("brainframe/cli/defaults.yaml")
```

Comment:
Which of these is preferred?
```python
RELATIVE_TRANSLATIONS_PATH = Path("brainframe/cli/translations")
RELATIVE_TRANSLATIONS_PATH = Path("brainframe", "cli", "translations")
RELATIVE_TRANSLATIONS_PATH = Path("brainframe") / "cli" / "translations"
```

Reply:
IMO my opinion, I like the first one best.